pipelines/jobs/configurations/jdk15_pipeline_config.groovy

```diff
@@ -14,7 +14,7 @@ class Config15 {
         x64Linux  : [
                 os                  : 'linux',
                 arch                : 'x64',
-                additionalNodeLabels: 'centos6',
+                additionalNodeLabels: 'rhel',
```

Comment:
probably needs to be `rhel7` to ensure that it's locked to 7.

Reply:
Cut & paster error - thanks :-) (`centos7||rhel7` would e best if it would work I guess, but the ones I have are rhel7 so that'll do for now)